### ISSUE TITLE
knative-eventing spam logs with `Failed to autoscale` records

### ISSUE THREAD

**Issue by omgbebebe** [mindwm/mindwm-gitops#69]
![image](https://github.com/user-attachments/assets/14f9b074-8c2b-4772-a04a-d4b012a86d61)

Loki request example
```
{service_name="knative-eventing"} |= `` | json | line_format `{{.log}}` | json | line_format `{{.level}}: {{.msg}}` | msg = `Failed to autoscale`
```

**Comment by omgbebebe** [comment]
the same for log records
```
{service_name="knative-eventing"} |= `` | json | line_format `{{.log}}` | json | line_format `{{.level}}: {{.msg}}` | msg =~ `error while refreshing scheduler state.*`
```

maybe it relates

**Comment by metacoma** [comment]
Fixed by #70
